docs/04.guides/04.cookbooks/24.Query-Handling/page.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Query Handling In Lucee
+id: query-handling-lucee
+---
+
+Here we can find how the lucee supports for SQL queries. 
+
+## Query tags ##
+
+```<cfquery>``` different ways to use the tags in lucee & how we can the passed the value into the query
+
+```lucee
+<cfquery name="qry" datasource="test">
+	select * from Foo1890
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+Above example just retrieve data from database.
```

Comment:
***toChange***
The above example just shows how to retrieve the data from the database.